addons/mail/static/src/components/call_main_view/call_main_view.js

```diff
@@ -17,6 +17,16 @@ export class CallMainView extends Component {
         useComponentToModel({ fieldName: 'component' });
         useRefToModel({ fieldName: 'tileContainerRef', refName: 'tileContainer', });
         useUpdateToModel({ methodName: 'onComponentUpdate' });
+
+        onMounted(() => {
+            this.resizeObserver = new ResizeObserver(() => {
+                this.callMainView.onResize();
+            });
+            this.resizeObserver.observe(this.root.el);
+        });
+        onWillUnmount(() => {
+            this.resizeObserver.disconnect();
+        });
```

Comment:
Ok for use of `ResizeObserver` as a fix here.
Could have less LOCs with inline arrow functions.